topology/University of Nebraska/Nebraska-CMS/Nebraska.yaml

```diff
@@ -303,35 +307,6 @@ Resources:
           endpoint: hcc-ps01.unl.edu
     VOOwnership:
       HCC: 100
-  US-Nebraska LT (Test):
-    Active: true
-    ContactLists:
-      Administrative Contact:
-        Primary:
-          ID: e86d60dc841731aebfd879d5883d66ebab20cbd2
-          Name: Garhan Attebury
-      Miscellaneous Contact:
-        Primary:
-          ID: e86d60dc841731aebfd879d5883d66ebab20cbd2
-          Name: Garhan Attebury
-      Resource Report Contact:
-        Primary:
-          ID: e86d60dc841731aebfd879d5883d66ebab20cbd2
-          Name: Garhan Attebury
-      Security Contact:
-        Primary:
-          ID: e86d60dc841731aebfd879d5883d66ebab20cbd2
-          Name: Garhan Attebury
-    Description: perfSONAR testbed latency host at the University of Nebraska-Lincoln.
-    FQDN: perfsonar.unl.edu
-    ID: 692
-    Services:
-      net.perfSONAR.Latency:
-        Description: PerfSonar Latency monitoring node
-        Details:
-          endpoint: perfsonar.unl.edu
-    VOOwnership:
-      HCC: 100
```

Comment:
@gattebury is there a good reason why there's separate resource entries for the bandwidth and latency monitors? Can we just merge the two since they're on the same host?

Reply:
There's no good reason I can think of. That host was originally in a pS development test mesh and there's a chance having separate entries made sense to someone, but that's in the past. Feel free to squash it down to a single entry.

Reply:
Will do, thanks for the quick response!